v2.0/configure-replication-zones.md

```diff
@@ -156,6 +156,23 @@ If you need to troubleshoot this command's behavior, you can change its [logging
 
 These examples focus on the basic approach and syntax for working with zone configuration. For examples demonstrating how to use constraints, see [Scenario-based Examples](#scenario-based-examples).
 
+###  View the Pre-Configured Replication Zones
```

Comment:
nit: I'd title this **List the Pre-Configured Replication Zones**, since we're not really viewing their configs.

Reply:
Done.